src/api/events.ts

```diff
@@ -0,0 +1,7 @@
+import { AxiosPromise } from 'axios';
```

Comment:
Is this intentional?

Reply:
good catch, wrong branch. Will remove it from here